apps/condo/domains/resident/tasks/sendBillingReceiptNotifications.task.js

```diff
@@ -1,14 +1,24 @@
-const { createCronTask } = require('@condo/keystone/tasks')
+const { createCronTask, removeCronTask } = require('@condo/keystone/tasks')
 
 const { sendBillingReceiptNotifications } = require('./helpers')
 
-const WORK_DAYS_FROM = 10
-const WORK_DAYS_TO = 15
-const WEEKENDS_FROM = 12
-const WEEKENDS_TO = 15
+/** PODS work according to UTC, so time in cron-tasks should also be according to UTC too. */
+const UTC_TO_MSC = -3
+const MSC_WORK_DAYS_FROM = 10
+const MSC_WORK_DAYS_TO = 15
+const MSC_WEEKENDS_FROM = 12
+const MSC_WEEKENDS_TO = 15
 
-const sendBillingReceiptNotificationsTaskWorkDays = createCronTask('sendBillingReceiptNotificationsTaskWorkDays', `17 ${WORK_DAYS_FROM}-${WORK_DAYS_TO} * * 1-5`, sendBillingReceiptNotifications)
-const sendBillingReceiptNotificationsTaskWeekends = createCronTask('sendBillingReceiptNotificationsTaskWeekends', `17 ${WEEKENDS_FROM}-${WEEKENDS_TO} * * 0,6`, sendBillingReceiptNotifications)
+const UTC_WORK_DAYS_FROM = MSC_WORK_DAYS_FROM + UTC_TO_MSC
+const UTC_WORK_DAYS_TO = MSC_WORK_DAYS_TO + UTC_TO_MSC
+const UTC_WEEKENDS_FROM = MSC_WEEKENDS_FROM + UTC_TO_MSC
+const UTC_WEEKENDS_TO = MSC_WEEKENDS_TO + UTC_TO_MSC
+
+removeCronTask('sendBillingReceiptNotificationsTaskWorkDays', `17 ${MSC_WORK_DAYS_FROM}-${MSC_WORK_DAYS_TO} * * 1-5`)
+removeCronTask('sendBillingReceiptNotificationsTaskWeekends', `17 ${MSC_WEEKENDS_FROM}-${MSC_WEEKENDS_TO} * * 0,6`)
+
+const sendBillingReceiptNotificationsTaskWorkDays = createCronTask('sendBillingReceiptNotificationsTaskWorkDays', `17 ${UTC_WORK_DAYS_FROM}-${UTC_WORK_DAYS_TO} * * 1-5`, sendBillingReceiptNotifications)
```

Comment:
Could you please HARDCODE the value. it's hard to read the real value! You don't need to reuse this consts >_<

Reply:
It's hard to check it